src/plugins/reporter-bugzilla.c

```diff
@@ -217,6 +217,17 @@ char *ask_bz_api_key(const char *message)
     return api_key;
 }
 
+
+static
+bool is_redhat_bugzilla(const char *bugzilla_url)
+{
+    if (g_str_has_suffix(bugzilla_url, "redhat.com")) {
```

Comment:
I suppose this could be just `return g_str_has_suffix(bugzilla_url, "redhat.com")` :slightly_smiling_face:

Reply:
Ah, good catch :)